src/System.Runtime/src/System/LoaderOptimizationAttribute.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    [AttributeUsage (AttributeTargets.Method)]
+    public sealed partial class LoaderOptimizationAttribute : System.Attribute
```

Comment:
Couple nits:
 - Does it need to be `partial` here?
 - `System.Attribute` => `Attribute`